index.js

```diff
@@ -0,0 +1,258 @@
+// @flow
+
+import GeocoderArcGIS from 'geocoder-arcgis'
+import lonlat from '@conveyal/lonlat'
+
+const arcGisGeocoders = {}
+
+type Boundary = {
+  rect: {
+    minLat: number,
+    minLon: number,
+    maxLat: number,
+    maxLon: number
+  }
+}
+
+type BaseQuery = {
+  clientId?: string,
+  clientSecret?: string,
+  url?: string
+}
+
+type BaseResponse = {
+  query: any
+}
+
+function boundaryToSearchExtent (boundary: Boundary): string {
+  return [
+    boundary.rect.minLon,
+    boundary.rect.maxLat,
+    boundary.rect.maxLon,
+    boundary.rect.minLat
+  ].join(',')
+}
+
+/**
+ * To make flow happy
+ */
+function coerceNullToString (s: ?string) {
```

Comment:
I've run into this too...are we doing something wrong? I feel like this is a common pattern.

Reply:
Looks like there has been a debate on this issue: https://github.com/facebook/flow/issues/2814  Maybe I should use toString() instead?